src/zenml/steps/step_environment.py

```diff
@@ -0,0 +1,74 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+from zenml.environment import BaseEnvironmentComponent
+
+STEP_ENVIRONMENT = "step_environment"
```

Comment:
When I first saw the import of that I wasn't quite sure what it was exactly, maybe `STEP_ENVIRONMENT_NAME` to make it a little clearer at first sight?

Reply:
done